src/components/settings/RemovePassphrasePrompt.tsx

```diff
@@ -1,17 +1,20 @@
 import React, { useEffect } from 'react';
-import { useDispatch } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import {
   Button,
   Dialog,
   DialogActions,
   DialogContent,
   DialogContentText,
   DialogTitle,
-  TextField
+  TextField,
+  Typography,
 } from '@material-ui/core';
 import { Trans } from '@lingui/macro';
-import { AlertDialog } from '@chia/core';
+import { AlertDialog, Flex } from '@chia/core';
 import { openDialog } from '../../modules/dialog';
+import TooltipIcon from '../core/components/TooltipIcon';
```

Comment:
you can import this component directly from '@chia/core'. ../core will be removed in the short run

Reply:
I'll update AppPassPrompt.tsx as well